packages/components/tree-v2/src/tree-node-content.vue

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import { h, defineComponent, inject } from 'vue'
+import { ROOT_TREE_INJECTION_KEY, treeNodeContentProps } from './virtual-tree'
+import type { RootTreeType } from './tree.type'
```

Comment:
We'd better to use variable names starting with the component name.
```suggestion
import type { TreeRootType } from './tree.type'
```